L.Map.Sync.js

```diff
@@ -83,19 +85,27 @@
         unsync: function (map) {
             var self = this;
 
+            if (this._cursors) {
+                this._cursors.forEach(function (cursor, indx, _cursors) {
+                    if (cursor === map.cursor) {
+                        _cursors.splice(indx, 1)
```

Comment:
yes, I agree. I also noticed some other oddities in unchanged lines, we should have a look at that. I can fix this line then too, so merging ;)